spec/supabase_py_v2.yml

```diff
@@ -1795,16 +1795,54 @@ functions:
         name: Basic invocation
         description:
         code: |
+          ```python
+          resp = supabase.functions.invoke(
+            "hello-world",
+            invoke_options={
+              "body": { "foo": "bar" }
+            },
+          )
           ```
-          func = supabase.functions()
-          @asyncio.coroutine
-          async def test_func(loop):
-            resp = await func.invoke("hello-world",invoke_options={'body':{}})
-            return resp
-
-          loop = asyncio.get_event_loop()
-          resp = loop.run_until_complete(test_func(loop))
-          loop.close()
+      - id: error-handling
+        name: Error handling
+        description: |
+          A `FunctionsHttpError` error is returned if your function throws an error, `FunctionsRelayError` if the Supabase Relay has an error processing your function.
+        isSpotlight: true
+        code: |
+          ```python
+          from supafunc.errors import FunctionsRelayError, FunctionsHttpError
```

Comment:
is `supafunc` something that we can change? Ideally we don't use things like `supaXX`. better if it can be `supabase-functions` - no abbreviations

Reply:
This is because the python package name is just that, I think `functions` was taken or even `functions-py` was taken too at the time of the packages being created. We could let the `supabase-py` package export these errors too but it would mean we flood that namespace with errors from every package.

```py
from supabase.errors import FunctionsRelayError, FunctionsHttpError
```

Reply:
@J0 how hard would it be to get the package renamed to `supabase-functions` on PyPi?

Reply:
> I think functions was taken or even functions-py was taken too at the time when the package was created. 
Yup, this is accurate - a poll was done on Slack and `supafunc` had the highest votes.

There's no concept of rename I think. We'll have to publish to the new namespace from here and deprecate the old namespace slowly.I've [created a new package](https://pypi.org/manage/project/supabase-functions/collaboration/) and sent out invites to both.